api/graphql/index.js

```diff
@@ -165,6 +165,12 @@ export function makeAuthenticatedQueries (userId, fetchOne, fetchMany) {
     post: (root, { id }) => fetchOne('Post', id),
     posts: (root, args) => fetchMany('Post', args),
     people: (root, args) => fetchMany('Person', args),
+    projects: (root, { context, contextId, viewerId }) => {
+      return Post.projectsForContext(context, contextId)
+        .then(({ models }) => {
+          return { items: models.filter(async p => {return await p.isVisibleToUser(p.id, viewerId) })}
```

Comment:
we dont otherwise do this type of filtering here, seems like this should happen in the projectsForContext function maybe?

Reply:
ok sounds good, I moved it into `projectsForContext`